src/test/java/teammates/test/cases/action/AdminEmailComposeSaveActionTest.java

```diff
@@ -52,7 +52,9 @@ public void testExecuteAndPostProcess() throws Exception {
                         Const.ParamsNames.ADMIN_EMAIL_ADDRESS_RECEIVERS, receiver);
         ShowPageResult pageResult = getShowPageResult(action);
         assertEquals(
-                Const.ViewURIs.ADMIN_EMAIL + "?error=false&user=admin.user", pageResult.getDestinationWithParams());
+                getPageResultDestination(
+                        Const.ViewURIs.ADMIN_EMAIL, false, adminUserId),
+                pageResult.getDestinationWithParams());
```

Comment:
Looks like putting `Const...` on the same line as `getPageResultDestination` would be fine (already done in this manner in other parts of the code), like so:
```java
assertEquals(
      getPageResultDestination(Const.ViewURIs.ADMIN_EMAIL, false, adminUserId),
      pageResult.getDestinationWithParams());
```

Leaving `Const...` and `pageResult...` on the same indentation level would be misleading, as `pageResult...` is not a parameter of `getPageResultDestination`.